### ISSUE TITLE
Disable animations with prefers-reduced-motion 

### ISSUE THREAD

**Issue by gusaiani** [metabase/metabase#18169]
## Is your feature request related to a problem? Please describe
For accessibility reasons, animations should be disabled (or very limited) to honor browser setting.
Would also make Cypress CI testing faster and sometimes avoid race-conditions, since it's waiting for animations to complete.
And is also helpful on older/slower computers, where CPU spikes when loading a dashboard with a handful of loading-spinners.

## Approach
Honor the browser setting for `prefers-reduced-motion`
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

## Additional context
General accessibility #1462

## Issues

* [x] #18041
* [x] #18082
* [x] #18098
* [x] #18102
* [x] #18172
* [x] #18174
* [x] #18189
* [x] #18191
* [ ] #18194
* [x] #18197
* [x] #18198
* [x] #18636
* [x] #18831

⚠️ List above is not exhaustive, please add issues as you find more cases worth treating.

**Comment by flamber** [comment]
Starting issue #17726

**Comment by alxnddr** [comment]
@gusaiani @flamber is there a plan to change our loader for those with `prefers-reduced-motion`? I personally use it and now the loader might look broken because it is the same loader but just without animation. I believe we need to have a specific static loader for this case. What do you think?

**Comment by flamber** [comment]
@alxnddr Firefox uses a static hourglass, which seems like a good substitute. Priority was to see the impact it would have, but that has been overall positive, so we can work on making it nicer now.

**Comment by gusaiani** [comment]
@alxnddr @flamber, thanks, issue created: #18191

**Comment by flamber** [comment]
Closing, since reduced motion has been implemented almost everywhere. Some arrived in 0.41.0, while most is in 0.42.0

**Comment by timuric** [comment]
@flamber I have enabled reduced motion, however animations are still there
![image](https://user-images.githubusercontent.com/4006792/143235232-aa9df973-d090-4cdd-9bd1-987fdd2e97ae.png)

I have tried chrome, safari and firefox. We are using Metabase cloud  ```v1.41.1 Built on 2021-10-21```

Also applying system wide settings just to be able to use Metabase is a bit overkill, would be great to be able to change this setting in the admin panel (so that other non technical users in the organization would also not have to deal with these issues).

**Comment by flamber** [comment]
@timuric This issue and most of the related issues are added to the upcoming 0.42
I don't know how Mac works. I use the configs in Firefox as noted in https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#user_preferences

This is an accessibility functionality. If you are looking for something else, then please open a new issue.

**Comment by ariya** [comment]
> We are using Metabase cloud `v1.41.1 Built on 2021-10-21`

Most of the disabled animations honoring reduced motion will be part of v42.

The only thing that was in v 41 is the sidebar animation, Filter and Summarize. Give it a try and see if that works in your browser.

![image](https://user-images.githubusercontent.com/7288/143294959-ff915248-60ef-4e4c-9508-99417656e2a6.png)
